aemedge/blocks/hero/hero.css

```diff
@@ -254,3 +258,11 @@ udex-hero-banner .text-neutral-black {
     --udexHeroBannerMinHeight: 450px;
   }
 }
+
+.hero-wrapper.full-background-image-style-wrapper udex-hero-banner {
+  --udexHeroBannerBackgroundImageObjectFit: cover;
+}
+
+.hero-container:has(.hero-dock-to-next-wrapper) {
```

Comment:
I don't understand `dock to next` not sure if authors' will. Why isn't this called "no bottom space" or similar?

Reply:
@mhaack : Let's ask editors what name works best for them. For me, "dock to next (element)" or "snap to next" made more sense than "no bottom space", because that could also be within the element (= align text to bottom)